app/views/sidebar/_related.html.erb

```diff
@@ -59,7 +59,7 @@
         <a href="/profile/<%= author.name %>"><%= author.name %></a>
       </p>
       <% end %>
-      <p><a href="/contributors/<%= params[:id] %>"><i class="fa fa-list"></i> <%= t('sidebar._related.view_all') %> &raquo;</a></p>
+      <p><a href="/contributors/<%= params[:id] %>"><i class="fa fa-list"></i> <%= t('sidebar._related.contributors_for_this_topic') %> &raquo;</a></p>
```

Comment:
This looks good but we need a matching locale file -- that'll provide the text here, like in this example:

https://github.com/publiclab/plots2/blob/8e1a7a02091781a5cf572236befeb79bc5292e35/config/locales/views/sidebar/_related/en.yml

Make sense? Thank you!!!